content/news/XXXX-XX-XX-mixxx-2_5-released.md

```diff
@@ -0,0 +1,564 @@
+  [#13869](https://github.com/mixxxdj/mixxx/issues/13869)
+  [#13873](https://github.com/mixxxdj/mixxx/pull/13873)
+  [#13875](https://github.com/mixxxdj/mixxx/pull/13875)
+  [#13898](https://github.com/mixxxdj/mixxx/pull/13898)
+  [#13899](https://github.com/mixxxdj/mixxx/pull/[#13899](https://github.com/mixxxdj/mixxx/pull/13899))
+* Update to latest vcpkg dependencies
+  [#11649](https://github.com/mixxxdj/mixxx/pull/11649)
+  [#12512](https://github.com/mixxxdj/mixxx/pull/12512)
+  [#12067](https://github.com/mixxxdj/mixxx/pull/12067)
+  [#12898](https://github.com/mixxxdj/mixxx/pull/12898)
+  [#13155](https://github.com/mixxxdj/mixxx/pull/13155)
+  [#13830](https://github.com/mixxxdj/mixxx/pull/13830)
+* GitHub actions updates
+  [#11544](https://github.com/mixxxdj/mixxx/pull/11544)
+  [#11508](https://github.com/mixxxdj/mixxx/pull/11508)
+  [#11487](https://github.com/mixxxdj/mixxx/pull/11487)
+  [#11438](https://github.com/mixxxdj/mixxx/pull/11438)
+  [#11410](https://github.com/mixxxdj/mixxx/pull/11410)
+  [#11560](https://github.com/mixxxdj/mixxx/pull/11560)
+  [#11578](https://github.com/mixxxdj/mixxx/pull/11578)
+  [#11610](https://github.com/mixxxdj/mixxx/pull/11610)
+  [#11631](https://github.com/mixxxdj/mixxx/pull/11631)
+  [#11710](https://github.com/mixxxdj/mixxx/pull/11710)
+  [#11736](https://github.com/mixxxdj/mixxx/pull/11736)
+  [#11920](https://github.com/mixxxdj/mixxx/pull/11920)
+  [#11961](https://github.com/mixxxdj/mixxx/pull/11961)
+  [#12241](https://github.com/mixxxdj/mixxx/pull/12241)
+  [#12394](https://github.com/mixxxdj/mixxx/pull/12394)
+  [#12447](https://github.com/mixxxdj/mixxx/pull/12447)
+  [#12425](https://github.com/mixxxdj/mixxx/pull/12425)
+  [#12421](https://github.com/mixxxdj/mixxx/pull/12421)
+  [#12799](https://github.com/mixxxdj/mixxx/pull/12799)
+  [#12801](https://github.com/mixxxdj/mixxx/pull/12801)
+  [#12800](https://github.com/mixxxdj/mixxx/pull/12800)
+  [#12736](https://github.com/mixxxdj/mixxx/pull/12736)
+  [#12692](https://github.com/mixxxdj/mixxx/pull/12692)
+  [#12694](https://github.com/mixxxdj/mixxx/pull/12694)
+  [#12695](https://github.com/mixxxdj/mixxx/pull/12695)
+  [#12691](https://github.com/mixxxdj/mixxx/pull/12691)
+  [#12693](https://github.com/mixxxdj/mixxx/pull/12693)
+  [#12625](https://github.com/mixxxdj/mixxx/pull/12625)
+  [#12627](https://github.com/mixxxdj/mixxx/pull/12627)
+  [#12626](https://github.com/mixxxdj/mixxx/pull/12626)
+  [#12577](https://github.com/mixxxdj/mixxx/pull/12577)
+  [#13162](https://github.com/mixxxdj/mixxx/pull/13162)
+  [#13163](https://github.com/mixxxdj/mixxx/pull/13163)
+  [#13187](https://github.com/mixxxdj/mixxx/pull/13187)
+  [#13217](https://github.com/mixxxdj/mixxx/pull/13217)
+  [#13246](https://github.com/mixxxdj/mixxx/pull/13246)
+  [#13232](https://github.com/mixxxdj/mixxx/pull/13232)
+  [#13528](https://github.com/mixxxdj/mixxx/pull/13528)
+  [#13595](https://github.com/mixxxdj/mixxx/pull/13595)
+

```

Comment:
```suggestion
```
This long list of issue IDs has no benefit for the user. It would be better to link https://github.com/mixxxdj/mixxx/blob/2.5.0/CHANGELOG.md instead

Reply:
woohoo